exec-unittests

```diff
@@ -102,25 +132,45 @@ _subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-sc
 let count=0
 for _major in ${major_list}; do
     if [[ "${_major}" == "agent" ]]; then
+        # The parallel program is really cool, but what a pain that it posts this
+        # citation request.
```

Comment:
Yikes!

> --citation
> Print the citation notice and BibTeX entry for GNU parallel, silence citation notice for all future runs, and exit. It will not run any commands.
> 
> If it is impossible for you to run --citation you can instead use --will-cite, which will run commands, but which will only silence the citation notice for this single run.
> 
> If you use --will-cite in scripts to be run by others you are making it harder for others to see the citation notice. The development of GNU parallel is indirectly financed through citations, so if your users do not know they should cite then you are making it harder to finance development. However, if you pay 10000 EUR, you have done your part to finance future development and should feel free to use --will-cite in scripts.
> 
> If you do not want to help financing future development by letting other users see the citation notice or by paying, then please consider using another tool instead of GNU parallel. You can find some of the alternatives in man parallel_alternatives.
> 

Can we use `--will-cite`??

Reply:
I feel like we should expand this code comment to point out that this usage of `parallel` is internal and automated; that we are not distributing product code which depends on this tool; and that we, as developers, have viewed the citation and therefore are justified suppressing future displays of it in our development processes.